specs/agents/tracing-instrumentation-db.md

```diff
@@ -69,6 +69,10 @@ The following fields are relevant for database and datastore spans. Where possib
 | __**service.target._**__ |<hr/>|<hr/>|
 |`_.type`| `dynamodb` ||
 |`_.name`| e.g. `us-east-1` | Use same value as `context.db.instance` |
+| __**otel.attributes._**__ |<hr/>|<hr/>|
+TODO
```

Comment:
Do you mean to have this `TODO` here?

Reply:
It was meant to stay there until we discussed in the weekly. But after double checking specs and the list of available commands I think the proposal, as a start, is good. the fields proposed are easy to get and provide valuable info to whom is using OTel attribs.

I'm going to remove it :)